MainTel/MainTel-Troubleshooting/MTCS-Troubleshooting/MTDome-Troubleshooting/MTDome-Recover-and-clear-Faults.rst

```diff
@@ -0,0 +1,122 @@
+.. Include one Primary Author and list of Contributors (comma separated) between the asterisks (*):
+.. |author| replace:: *I. Sotuela*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *K. Peña*, *P. Venegas*
```

Comment:
Hi Paulina,
Thank you for the document. Please find below my comments. I only have one major comment: the [original procedure](https://confluence.lsstcorp.org/display/OOD/MTDome+Recover+and+clear+Faults) mentions in the comments, that the resetDrivesAz is not required. In your PR you keep it. Also the stop and engage breaks SAL script is in the original procedure but not here. Not sure which version is the one that is correct. Finally, Wouter also mentions a set_summary_state that is not in the document.